.github/workflows/add-new-impl-versions.yml

```diff
@@ -27,7 +27,8 @@ jobs:
         uses: actions/checkout@v3
       - name: Configure git
         run: |
-          git fetch $BRANCH && git checkout $BRANCH && git rebase $GITHUB_REF -X theirs || git checkout -b $BRANCH
+          git fetch origin $BRANCH && git checkout $BRANCH || git checkout -b $BRANCH
+          git rebase $GITHUB_REF -X theirs || git rebase --abort
```

Comment:
This fixes checkout which was previously always failing, and rebase so that we can get out of a situation where we cannot rebase cleanly.